Project doesn't build

On checking out master, doing a `dotnet build` gives the following errors:

```
error MSB4184: The expression "registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder" cannot be evaluated.
warning NU1503: Skipping restore for project 'C:\Users\Isaac\Source\Repos\EFCore.FSharp\EFCore.FSharp\EFCore.FSharp.fsproj'. The project file may be invalid or missing targets required for restore. [C:\Users\Isaac\Source\Repos\EFCore.FSharp\EFCore.FSharp.sln]
error MSB4184: The expression "registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder" cannot be evaluated.

Restore completed in 46.54 ms for C:\Users\Isaac\Source\Repos\EFCore.FSharp\EFCore.FSharp.Test\EFCore.FSharp.Test.fsproj.

error MSB4184: The expression "registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder" cannot be evaluated.
error MSB4184: The expression "registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder" cannot be evaluated.
```

How can I fix this?

--- a/EFCore.FSharp/EFCore.FSharp.fsproj
+++ b/EFCore.FSharp/EFCore.FSharp.fsproj
@@ -16,11 +16,7 @@
     <AssemblyOriginatorKeyFile>key.snk</AssemblyOriginatorKeyFile>
     <LangVersion>latest</LangVersion>
     <DocumentationFile>bin\$(Configuration)\$(TargetFramework)\$(AssemblyName).xml</DocumentationFile>
-    <TimestampUrl>http://time.certum.pl</TimestampUrl>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
-    <WindowsSdkDir Condition=" '$(WindowsSdkDir)' == '' ">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder)</WindowsSdkDir>
-    <WindowsSDKVersion Condition=" '$(WindowsSDKVersion)' == '' ">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@ProductVersion)\</WindowsSDKVersion>
-    <SignToolCommand>"$(WindowsSdkDir)bin\$(WindowsSDKVersion)x86\signtool.exe" sign /t $(TimestampUrl) /q</SignToolCommand>
     <Version>1.0.0-beta1</Version>
   </PropertyGroup>
   <ItemGroup>
@@ -36,6 +32,12 @@
     </None>
   </ItemGroup>
   <Target Name="SignOutput" AfterTargets="Build" Condition="'$(SignOutput)' == 'True'">
+    <PropertyGroup>
+      <TimestampUrl>http://time.certum.pl</TimestampUrl>
+      <WindowsSdkDir Condition=" '$(WindowsSdkDir)' == '' ">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@InstallationFolder)</WindowsSdkDir>
+      <WindowsSDKVersion Condition=" '$(WindowsSDKVersion)' == '' ">$(registry:HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Microsoft SDKs\Windows\v10.0@ProductVersion)\</WindowsSDKVersion>
+      <SignToolCommand>"$(WindowsSdkDir)bin\$(WindowsSDKVersion)x86\signtool.exe" sign /t $(TimestampUrl) /q</SignToolCommand>
+    </PropertyGroup>
     <Exec Command="$(SignToolCommand) &quot;$(TargetPath)&quot;" />
   </Target>
   <Import Project="..\.paket\Paket.Restore.targets" />

